modules/mono/glue/GodotSharp/GodotSharp/Core/StringExtensions.cs

```diff
@@ -1141,7 +1149,7 @@ private static bool WildcardMatch(ReadOnlySpan<char> str, ReadOnlySpan<char> pat
                     return WildcardMatch(str, pattern.Slice(1), caseSensitive)
                         || (!str.IsEmpty && WildcardMatch(str.Slice(1), pattern, caseSensitive));
                 case '?':
-                    return !str.IsEmpty && str[0] != '.' &&
+                    return !str.IsEmpty && str[0] is not '.' &&
```

Comment:
There are a lot of changes from `==` to `is` and `!=` to `is not` that are just noise.